Modules/readline.c

```diff
@@ -151,6 +151,24 @@ decode(const char *s)
 }
 
 
+/*
+   Explicitly disable bracketed paste mode. This is called both during module
+   initialization and after reading the init file. Even if the version of
+   Readline enables it by default or a user has it set, the Python REPL doesn't
+   support bracketed paste and it injects escape sequences which cause test
+   failures. This keeps the behavior consistent with user expectations for the
+   Python REPL when pasting multi-line strings. See bpo-42819 for more details.
```

Comment:
I suggest to copy/paste the NEWS entry here, I prefer it over this comment which is similar but different :-) (just remove "readline: " prefix and "Patch by ...".)

Keep the last sentence, but just add a new line between the two ;-)